demo/vl-proza-message.html

```diff
@@ -113,11 +113,11 @@ <h2>Proza message</h2>
         <h2>Proza message waarvan updaten misgaat</h2>
         <pre>
             <code class="line-numbers language-markup">
-                &lt;vl-proza-message id=&quot;message-1&quot; data-vl-domain=&quot;foo&quot; data-vl-code=&quot;fout&quot;&gt;&lt;/vl-proza-message&gt;
+                &lt;vl-proza-message data-vl-domain=&quot;foo&quot; data-vl-code=&quot;fout&quot;&gt;&lt;/vl-proza-message&gt;
             </code>
         </pre>
         <div class="demo">
-            <vl-proza-message id="message-1" data-vl-domain="foo" data-vl-code="fout"></vl-proza-message>
+            <vl-proza-message id="message-2" data-vl-domain="foo" data-vl-code="fout"></vl-proza-message>
```

Comment:
als ik al nummerkes in mijn ids zou gebruiken zou ik zien da ze "kloppen" ...
of zit er hier nu ene voor da ik nie zie in deze review?

Reply:
Puur gemakzucht.